packages/core/utils/src/common/define-config.ts

```diff
@@ -117,6 +117,13 @@ export function defineConfig(config: Partial<ConfigModule> = {}): ConfigModule {
     ...config.modules,
   }
 
+  // Remove any modules set to false
+  Object.keys(modules).forEach((key) => {
+    if (modules[key] === false) {
+      delete modules[key]
+    }
+  })
```

Comment:
Q: should we be able to disabled modules that have a default package as well?

Reply:
yes - e.g., in platform we don't want stock/inventory/etc. 

it will break functionality in the core API because those modules aren't there, but if you want to use medusa partially you should be able to config your way to that.